src/state/slices/tradeQuoteSlice/tradeQuoteSlice.ts

```diff
@@ -44,34 +54,77 @@ export const tradeQuoteSlice = createSlice({
     },
     setConfirmedQuote: (state, action: PayloadAction<TradeQuote | undefined>) => {
       state.confirmedQuote = action.payload
+      state.tradeExecution = initialTradeExecutionState
     },
     resetConfirmedQuote: state => {
       state.confirmedQuote = undefined
+      state.tradeExecution = initialTradeExecutionState
```

Comment:
Will we ever have a state where we want to reset only the `tradeExecution` slice bit, but not the `confirmedQuote`?